tests/testing_data/inference.json

```diff
@@ -1,5 +1,5 @@
 {
-    "device": "$torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')",
+    "device": "$torch.device('cuda' if torch.cuda.is_available() else 'cpu')",
```

Comment:
The previous version will avoid unnecessary allocation of multi gpu memories, could we keep it?

Reply:
updated in the latest commit.

Thanks.